### PR TITLE
Revised grid layout

### PR DESCRIPTION
**Pull request by mozartfish**
Tasks Completed (currently works for les mis and flight datasets)
- resized the cells to 11 x 11 
- resized grid for the new cells
- reshaped cells to have round corners
- fixed the row and column highlights to work with the new cell and grid size

Things that might need fixing
- resizing the labels (don't know if they are too big with the new layout)

### CODE REVIEW

File: src/components/AdjMatrix/AdjMatrixMethods.ts [hunk 1]
```diff
@@ -276,14 +282,17 @@ export class View {
 
     this.drawGridLines();
 
-    // add the highlight rows
+    // set the size of the highlight
+    const matrixHighlight = this.network.nodes.length * cellSize;
```
Comment (JackWilb) [inline src/components/AdjMatrix/AdjMatrixMethods.ts:286]: If you define this earlier you can reduce redundancy on 261 by using this var there

Comment (JackWilb) [reply src/components/AdjMatrix/AdjMatrixMethods.ts:286]: I think you should use a different, more descriptive, name here too. E.g. `matrixHighlightLength`

Comment (mozartfish) [reply src/components/AdjMatrix/AdjMatrixMethods.ts]: updated the code to reflect the changes you requested

**Comment by mozartfish** [comment]
fixed the items that needed to be changed and started a new branch implementing Roni's suggestion

**Pull request merged**